src/dataregistry/schema/schema.yaml

```diff
@@ -436,11 +453,11 @@ tables:
         description: "Date the dataset was last moved"
       location_type:
         type: "String"
-        choices: ["dataregistry", "external", "dummy"]
+        choices: ["dataregistry", "external", "meta_only",  "dummy"]
         cli_default: "dataregistry"
         cli_optional: True
         nullable: False
-        description: "What is the physical location of the data? 'dataregistry' means the data is located within the `<root_dir>` and managed by the `dataregistry`, external means the data is not managed by the dataregistry, either because it is off-site or because it is stored outside <root_dir> therefore there is only a database entry (in this case a `url` or `contact_email` must be provided during registration) and 'dummy' is a dataset for testing purposes only (only a database entry is created in this case)."
+        description: "What is the physical location of the data? 'dataregistry' means the data is located within the `<root_dir>` and managed by the `dataregistry`, external means the data is not managed by the dataregistry, either because it is off-site or because it is stored outside <root_dir> therefore there is only a database entry (in this case a `url` or `contact_email` must be provided during registration), 'meta_only' is for a legitimate entry involving no actual data and 'dummy' is a dataset for internal testing purposes only. The data registry will only attempt to manage data created with this field set to 'dataregistry'."
```

Comment:
meta_only and external seem the same from this description. maybe an "e.g., a gcrcatalogs entry"?

Reply:
Will update the description in next commit.